doodler/components.py

```diff
@@ -39,7 +42,7 @@ class Toggle(pn.reactive.ReactiveHTML):
     color = param.String(doc='Color of the border in hex format')
 
     _template = """
-    <button id="button" style="text-decoration: {{ 'underline' if active else 'normal' }};border-color:{{ color }};border-width:4px;border-radius:5%;padding:10px;font-weight:{{ 'bold' if active else 'normal' }}" onclick="${_update}">
```

Comment:
It looks like something went wrong when you merged/rebased your branch onto master, this should not be changed by your PR I think.

Reply:
Agreed. Sorry.